### ISSUE TITLE
Argo server is breaking intermittently

### ISSUE THREAD

**Issue by chittu11** [argoproj/argo#2954]
Hi,
We deployed argo 2.6.2 and its working fine. But we have to run "argo server" command in the background in order to see the argo server UI. sometimes the background process is breaking and its causing issues with argo servers and we are unable to access the UI as well as our job submission is getting crippled. I deployed argo server using the following instruction from official argo github:

https://github.com/argoproj/argo/tree/master/manifests/base/argo-server

Is there anyway I can have "argo server" service running continuously without initiating it manually through command line in my shell like "argo server"?

```
argo: v2.6.2
BuildDate: 2020-03-12T16:04:25Z
GitCommit: be0a0bb
GitTreeState: clean
GitTag: v2.6.2
GoVersion: go1.13.4
Compiler: gc
Platform: linux/amd64
```

Kubernetes version:

```
Client Version: v1.18.2
Server Version: v1.17.1
```

Thank you and appreciate any help or inputs with this issue?

Regards,
CS

**Comment by simster7** [comment]
How is this any different than https://github.com/argoproj/argo/issues/2945?

**Comment by chittu11** [comment]
Apologies. I saw your comment there. thank you. will give a try.

**Comment by simster7** [comment]
Closed as duplicate of #2945
